### PR TITLE
New member Chris

### PR DESCRIPTION
**Pull request by chrishan17**

### CODE REVIEW

**Comment by hgl** [comment]
Nice gif, you are putting pressure on me man. :D

**Comment by dfguo** [comment]
@threegoldstone can you update the PR?

**Comment by chrishan17** [comment]
already updated~ @dfguo

**Comment by dfguo** [comment]
reviewed

File: source/_posts/Chris-Han-Joins-Strikingly.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: Chris Han Joins Strikingly
+date: 2015-09-06 19:57:15
+tags:
+categories:
+- Welcome
```
Comment (dfguo) [inline source/_posts/Chris-Han-Joins-Strikingly.md]: use Team category

File: source/_posts/Chris-Han-Joins-Strikingly.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Frontend
```
Comment (dfguo) [inline source/_posts/Chris-Han-Joins-Strikingly.md]: this can be removed

**Comment by chrishan17** [comment]
@dfguo  updated

**Pull request merged**